code/turf/turf.dm

```diff
@@ -851,6 +856,8 @@ proc/generate_space_color()
 		if (map_settings.auto_windows)
 			for (var/obj/window/auto/W in orange(1))
 				W.UpdateIcon()
+	for (var/obj/ladder/embed/L in orange(1))
+		L.UpdateIcon()
```

Comment:
Why are these updates only in these specific rarely used procs and not in base `ReplaceWith`? Shouldn't the ladder update when the turf is updated in any way?

(also as a sidenote, we really should do this stuff in some better way, signals sent on turf updates; but out of the scope of this PR)